lib/dao.js

```diff
@@ -67,9 +67,8 @@ function convertSubsetOfPropertiesByType(inst, data) {
   for (var key in data) {
     // Convert the properties by type
     typedData[key] = inst[key];
-    if (typeof typedData[key] === 'object'
-            && typedData[key] !== null
-            && typeof typedData[key].toObject === 'function') {
+    if (typeof typedData[key] === 'object' && typedData[key] !== null &&
+        typeof typedData[key].toObject === 'function') {
```

Comment:
One expression per line please. We don't spell it explicitly in our style guide yet, but the examples are already following that rule.

https://github.com/strongloop-internal/loopback-knowledge-base/blob/master/style-guide.md#indentation-of-multi-line-expressions-in-if